airflow/config_templates/config.yml

```diff
@@ -2591,3 +2591,23 @@ sensors:
       type: float
       example: ~
       default: "604800"
+scarf_analytics:
+  description: |
+    Airflow integrates `Scarf <https://about.scarf.sh/>`__ to collect basic telemetry data during operation.
+    This data assists Airflow maintainers in better understanding how Airflow is used.
+    Insights gained from this telemetry are critical for prioritizing patches, minor releases, and
+    security fixes. Additionally, this information supports key decisions related to the development roadmap.
```

Comment:
Same like in `docs/apache-airflow/faq.rst` you are not listing _what_ is going to be collected. In regards of transparency I'd propose we name it here. Otherwise if you fear redundancy, I'd propose to point to docs of FAQ in `docs/apache-airflow/faq.rst`

Reply:
Updated